tailcfg/tailcfg.go

```diff
@@ -2455,6 +2455,25 @@ type HealthChangeRequest struct {
 	NodeKey key.NodePublic
 }
 
+// SetDeviceAttributesRequest is a request to update the
+// current node's attributes.
+//
+// As of 2024-12-30, this is an experimental dev feature
+// for internal testing. See tailscale/corp#24690.
+type SetDeviceAttributesRequest struct {
+	Version CapabilityVersion
+	NodeKey key.NodePublic
```

Comment:
Might be good to say what the semantics are for how it gets used / how authentication works?

Reply:
It's identical to how all the machine requests work (with a Version and then a NodeKey that must match the machine key used for the ts2021 transport) but I can say a bit more.

Reply:
done